lidiabrowser/sync/populate.py

```diff
@@ -17,11 +20,49 @@
     Publication,
     TermGroup,
 )
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+
 
 logger = logging.getLogger(__name__)
 
 
 LIDIAPREFIX = "~~~~LIDIA~~~~"
+LEXICON_URLS = {}
+
+
+def fetch_lexicon_data():
+    url = "https://github.com/CentreForDigitalHumanities/lidia-zotero/raw/main/vocabulary/lexicon.xlsx"
+    filename = "../data/lexicon.xlsx"
```

Comment:
This relies on the current directory, which may not work on the server because of permissions. I think it can be replaced by `settings.BASE_DIR / "data" / "lexicon.xlsx"`, but data dir should probably also stored in `settings.py`